src/main/kotlin/net/maizegenetics/phgv2/cli/AlignAssemblies.kt

```diff
@@ -0,0 +1,320 @@
+package net.maizegenetics.phgv2.cli
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.int
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.withContext
+import org.apache.logging.log4j.LogManager
+import java.io.File
+import java.util.stream.Collectors
+
+/**
+ * This will align assemblies to a reference genome.
```

Comment:
Expand this comment, mentioning anchorwave , and that is is using anchorwave's proali to align, which handles genome alignment with relocation variation, chromosome fusion or whole genome duplication.

Reply:
Ok, added